project/Dependencies.scala

```diff
@@ -175,7 +175,7 @@ object Dependencies {
   )
 
   private[this] val protoSchemasLib =
-    "com.wavesplatform" % "protobuf-schemas" % "1.2.4" classifier "proto"
+    "com.wavesplatform" % "protobuf-schemas" % "1.2.5-SC-545-SNAPSHOT" classifier "proto"
```

Comment:
should be switched to master version I guess?

Reply:
Waiting
https://github.com/wavesplatform/protobuf-schemas/pull/26